src/Features/CSharp/Portable/Completion/CompletionProviders/ExplicitInterfaceTypeCompletionProvider.cs

```diff
@@ -121,23 +121,23 @@ private bool IsPreviousTokenValid(SyntaxToken tokenBeforeType)
         {
             if (tokenBeforeType.Kind() == SyntaxKind.OpenBraceToken)
             {
-                // Show us after the open brace for a class/struct
-                return IsClassOrStruct(tokenBeforeType.Parent);
+                // Show us after the open brace for a class/struct/interface
+                return IsClassOrStructOrInterface(tokenBeforeType.Parent);
             }
 
             if (tokenBeforeType.Kind() == SyntaxKind.CloseBraceToken ||
                 tokenBeforeType.Kind() == SyntaxKind.SemicolonToken)
             {
-                // Check that we're after a class/struct member.
+                // Check that we're after a class/struct/interface member.
                 var memberDeclaration = tokenBeforeType.GetAncestor<MemberDeclarationSyntax>();
                 return memberDeclaration?.GetLastToken() == tokenBeforeType &&
-                       IsClassOrStruct(memberDeclaration.Parent);
+                       IsClassOrStructOrInterface(memberDeclaration.Parent);
             }
 
             return false;
         }
 
-        private static bool IsClassOrStruct(SyntaxNode node)
-            => node.Kind() == SyntaxKind.ClassDeclaration || node.Kind() == SyntaxKind.StructDeclaration;
+        private static bool IsClassOrStructOrInterface(SyntaxNode node)
+            => node.Kind() == SyntaxKind.ClassDeclaration || node.Kind() == SyntaxKind.StructDeclaration || node.Kind() == SyntaxKind.InterfaceDeclaration;
```

Comment:
>node.Kind() [](start = 15, length = 11)

nit: I think you could do `=> node.IsKind(SyntaxKind.ClassDeclaration, SyntaxKind.StructDeclaration, SyntaxKind.InterfaceDeclaration);`